cli/src/command.rs

```diff
@@ -288,16 +260,13 @@ pub fn run() -> sc_cli::Result<()> {
 
 			if chain_spec.is_crab() {
 				runner.sync_run(|config| {
-					// <--- dvm ---
 					// Remove dvm offchain db
-					let dvm_database_config = sc_service::DatabaseConfig::RocksDb {
-						path: crab_service::dvm_database_dir(&config),
+					let dvm_database_config = DatabaseSource::RocksDb {
+						path: darwinia_node_service::dvm::db_path(&config),
```

Comment:
Darwinia and Crab use the same darwinia_node_service crate, right?

Reply:
Yes. This is a node service crate.